searx/webapp.py

```diff
@@ -120,6 +120,7 @@
 # renaming names from searx imports ...
 from searx.autocomplete import search_autocomplete, backends as autocomplete_backends
 from searx.languages import language_codes as languages
+from searx.shared.redisdb import initialize as redis_initialize
```

Comment:
Instead of having different names for the same object (``import .. as ..``) I would prefer:

```python
from searx.shared import redisdb
...
redisdb.initialize()
```

this will also reduce the imported names (if more names from that imported lib are needed).